configure

```diff
@@ -2056,6 +2057,29 @@ sub get_features {
   return %features;
 }
 
+sub write_opendds_mwc {
+  my $root = shift;
+  my $args = shift;
+
+  $args = join(', ', (map {perlstring($_)} @{$args}));
+
+  my $path = "$root/bin/opendds_mwc.pl";
+  my $fd = backup_and_open($path);
+  print $fd <<"EOF";
+#!/usr/bin/env perl
+if (!defined(\$ENV{ACE_ROOT}) || !defined(\$MPC_ROOT)) {
+  die("The enviroment needs to be setup.");
+}
+exec('perl', "\$ENV{ACE_ROOT}/bin/mwc.pl", $args, \@ARGV)
+  or die("Failed to exec mwc: \$!");
+EOF
+  $fd->close();
+  print "Wrote $path\n" if $opts{'verbose'};
+  dump_and_unlink($fd) if $opts{'dry-run'};
```

Comment:
If `opendds_mwc.pl` already exists and then `configure` is run with the `dry-run` flag... it looks like this makes a backup but deletes the old one? Is that the desired behavior? I guess theoretically you shouldn't be re-running configure, even with `dry-run` but it seems like you'd want to preserve the existing `opendds_mwc.pl` in that case.

Reply:
If it's dry run, then the file descriptor it's unlinking inside `dump_and_unlink` is for a temporary file: https://github.com/objectcomputing/OpenDDS/blob/e1d588f4559a2dfedf817c9844e132dfea00fad6/configure#L1022-L1024